### PR TITLE
[full-ci] [tests-only] Remove `isTestingOnOcisOrReva` method

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
This PR removes the method `isTestingOnOcisOrReva` and refactor code where it is used.

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/5337

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/TestHelpers/OcisHelper.php [hunk 1]
```diff
@@ -47,18 +47,11 @@ public static function isTestingOnReva():bool {
...
-		return (self::isTestingOnOcis() || self::isTestingOnReva());
-	}
-
 	/**
 	 * @return bool
 	 */
 	public static function isTestingOnOc10():bool {
-		return (!self::isTestingOnOcisOrReva());
+		return false;
 	}
```
Comment (SagarGi) [inline tests/TestHelpers/OcisHelper.php:55]: is this function used anywhereelse in `ocis`?

Comment (grgprarup) [reply tests/TestHelpers/OcisHelper.php:55]: @grgprarup This PR is just to remove `isTestingOnOcisOrReva` method and its use. There are lots of things to remove and will do in other PRs.

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 2]
```diff
@@ -1134,33 +1093,14 @@ public function usersHaveBeenCreated(
...
-				$editData,
-				$this->getAdminUsername(),
-				$this->getAdminPassword(),
-				$this->stepLineRef
+				$userAttributes['userid'],
+				$userAttributes['password'],
+				$this->getStepLineRef()
 			);
+			// We don't need to set displayName and email while running in oCIS
+				// As they are set when creating the user
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/Provisioning.php:1103]:
```suggestion
```

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 3]
```diff
@@ -1587,20 +1516,15 @@ public function theAdministratorSendsAUserCreationRequestForTheFollowingUsersWit
...
-			$bodyTable = new TableNode(
-				[
+		$email = $userToCreate . '@owncloud.com';
+		$bodyTable = new TableNode(
+			[
 					['userid', $userToCreate],
 					['password', $password],
 					['username', $userToCreate],
 					['email', $email]
 				]
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/Provisioning.php:1526]:
```suggestion
			[
				['userid', $userToCreate],
				['password', $password],
				['username', $userToCreate],
				['email', $email]
			]
```

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 4]
```diff
@@ -1634,23 +1558,16 @@ public function theAdministratorCreatesUserPasswordGroupUsingTheProvisioningApi(
...
-				[
+		$email = $user . '@owncloud.com';
+		$bodyTable = new TableNode(
+			[
 					['userid', $user],
 					['password', $password],
 					['username', $user],
 					['email', $email],
 					['groups[]', $group],
 				]
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/Provisioning.php:1569]:
```suggestion
			[
				['userid', $user],
				['password', $password],
				['username', $user],
				['email', $email],
				['groups[]', $group],
			]
```

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 5]
```diff
@@ -1689,23 +1604,16 @@ public function theGroupAdminCreatesUserPasswordGroupUsingTheProvisioningApi(
...
-				[
+		$email = $userToCreate . '@owncloud.com';
+		$bodyTable = new TableNode(
+			[
 					['userid', $userToCreate],
 					['password', $userToCreate],
 					['username', $userToCreate],
 					['email', $email],
 					['groups[]', $group],
 				]
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/Provisioning.php:1615]:
```suggestion
			[
				['userid', $userToCreate],
				['password', $userToCreate],
				['username', $userToCreate],
				['email', $email],
				['groups[]', $group],
			]
```

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 6]
```diff
@@ -1743,23 +1649,16 @@ public function theGroupAdminCreatesUserPasswordInOtherGroupUsingTheProvisioning
...
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/Provisioning.php:1660]:
```suggestion
			[
				['userid', $userToCreate],
				['password', $userToCreate],
				['username', $userToCreate],
				['email', $email],
				['groups[]', $group],
			]
```

**Review by SagarGi — COMMENTED** [review]
Other Looks good to me :+1:

File: tests/acceptance/features/bootstrap/Provisioning.php [hunk 7]
```diff
@@ -1515,20 +1449,15 @@ public function adminSendsUserCreationRequestWithFollowingAttributesUsingTheProv
...
-			$bodyTable = new TableNode(
-				[
+		$email = $user . '@owncloud.com';
+		$bodyTable = new TableNode(
+			[
 					['userid', $user],
 					['password', $password],
 					['username', $user],
 					['email', $email]
 				]
```
[the remark this answers is not in the document]
Comment (grgprarup) [reply tests/acceptance/features/bootstrap/Provisioning.php:1459]: I am surprised why `test-php-style-fix` does not fix these kinds of formatting.

**Review by saw-jan — APPROVED** [review]
LGTM

File: tests/TestHelpers/LoggingHelper.php [hunk 8]
```diff
@@ -53,32 +53,8 @@ class LoggingHelper {
 	public static function getLogFilePath(
 		?string $xRequestId = ''
 	):string {
-		if (OcisHelper::isTestingOnOcisOrReva()) {
-			// Currently we don't interact with the log file on reva or OCIS
-			return "";
-		}
```
Comment (SwikritiT) [inline tests/TestHelpers/LoggingHelper.php:59]: So this whole function can be removed?

File: tests/TestHelpers/LoggingHelper.php [hunk 9]
```diff
@@ -163,23 +123,8 @@ public static function setLogLevel(
 		?string $logLevel,
 		?string $xRequestId = ''
 	):void {
-		if (OcisHelper::isTestingOnOcisOrReva()) {
-			// Currently we can't manage log file settings on reva or OCIS
-			return;
-		}
-		if (!\in_array($logLevel, self::LOG_LEVEL_ARRAY)) {
```
Comment (SwikritiT) [inline tests/TestHelpers/LoggingHelper.php:170]: This function too but can be done in next PR

**Review by SwikritiT — APPROVED** [review]
LGTM

File: tests/TestHelpers/LoggingHelper.php (hunk 8 above)
Comment (grgprarup) [reply tests/TestHelpers/LoggingHelper.php:59]: Will do in another PR.

**Pull request merged**